src/Options.hs

```diff
@@ -67,22 +71,34 @@ data Run = Run {
 , runOptions :: [String]
 , runMagicMode :: Bool
 , runFastMode :: Bool
+, runPreserveIt :: Bool
 } deriving (Eq, Show)
 
+defaultMagic, defaultFastMode, defaultPreserveIt :: Bool
```

Comment:
To clarify, defining the types of multiple bindings in one line might be convenient for e.g. writing papers, but I personally prefer not to use this particular syntactic feature in real-world code.